nhais-adaptor/adaptor/outgoing/fhir_helpers/operation_definition.py

```diff
@@ -0,0 +1,166 @@
+             'profile': {'reference': f"#{reference}"}})
+        return op_param
+
+    @staticmethod
+    def create_operation_definition(name, code, date_time, contained, parameter):
+        """
+        Helper function to create a fhir operation definition
+        :param name: The name of the operation
+        :param code: The code of the operation
+        :param date_time: the time stamp of the operation
+        :param contained: list of contained resources
+        :param parameter: list of parameters
+        :return: OperationDefinition
+        """
+        op = OperationDefinition(
+            {'name': name,
+             'kind': 'operation',
+             'type': True,
+             'system': False,
+             'instance': False,
+             'date': date_time,
+             'status': 'active',
+             'code': code})
+        if len(contained) > 0:
+            op.contained = contained
+        if len(parameter) > 0:
+            op.parameter = parameter
+        return op
+
+    @staticmethod
+    def create_practitioner_resource(resource_id, national_identifier, local_identifier):
+        """
+        Helper function to create a fhir Practitioner resource
+        :param resource_id: the id of the resource
+        :param national_identifier: the national identifier
+        :param local_identifier: the local identifier
+        :return: Practitioner
+        """
+        practitioner = Practitioner({
+            'id': resource_id,
+            'identifier': [
+                {
+                    'type': {
+                        'coding': [{'code': 'national'}],
+                        'text': 'GMC National code'
+                    },
+                    'value': national_identifier
+                },
+                {
+                    'type': {
+                        'coding': [{'code': 'local'}],
+                        'text': 'Local GP Code'
+                    },
+                    'value': local_identifier
+                }
+            ]
+        })
+        return practitioner
+
+    @staticmethod
+    def create_patient_resource(resource_id, nhs_number, title, first_name, last_name, gender, date_of_birth,
+                                address_line_1, city, postcode, place_of_birth):
+        """
+        Helper function to create a basic fhir patient
+        :param resource_id: the id of the resource
+        :param nhs_number: the nhs number of the patient
+        :param title: title of the patient
+        :param first_name: first name of the patient
+        :param last_name: last name of the patient
+        :param gender: patient gender
+        :param date_of_birth: date of birth of patient
+        :param address_line_1: first line of address
+        :param city: city of patient
+        :param postcode: post code of patient
+        :param place_of_birth: place of birth of patient
+        :return: Patient
+        """

```

Comment:
As discussed, we suggested remove trivial documentation and try to keep to docs that explain an unclear variable name, return type or action.

Reply:
I agree, the documentation here seems unnecessary 👍